libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/NavigationCamera.java

```diff
@@ -220,7 +220,7 @@ public void resetCameraPositionWith(@TrackingMode int trackingMode) {
    * @deprecated use {@link #showRouteGeometryOverview(int[])} instead
    */
   @Deprecated
-  public void showRouteOverview(@NonNull int[] padding) {
+  public void showOverviewForRoute(@NonNull int[] padding) {
```

Comment:
This is a semver-major change. We should live the deprecated API as-is.

Reply:
yeah, that was changed by mistake.